src/SFA.DAS.AssessorService.Application/Handlers/Staff/StartCertificateHandler.cs

```diff
@@ -43,26 +43,28 @@ public class StartCertificateHandler : IRequestHandler<StartCertificateRequest,
 
         public async Task<Certificate> Handle(StartCertificateRequest request, CancellationToken cancellationToken)
         {
+            _logger.LogInformation($"StartCertificateRequest for EPAO Ukprn: {request.UkPrn}");
+            var organisation = await _organisationQueryRepository.GetByUkPrn(request.UkPrn);
             var certificate = await _certificateRepository.GetCertificate(request.Uln, request.StandardCode);
 
             if (certificate == null)
-                certificate = await CreateNewCertificate(request);
+                certificate = await CreateNewCertificate(request, organisation);
             else
             {
                 var certData = JsonConvert.DeserializeObject<CertificateData>(certificate.CertificateData);
-
-                _logger.LogInformation("CreateNewCertificate Before Get Ilr from db");
                 var ilr = await _ilrRepository.Get(request.Uln, request.StandardCode);
 
                 if (certificate.Status == CertificateStatus.Deleted && ilr != null)
                 {
+                    _logger.LogInformation($"Recreating deleted certificate for ULN: {certificate.Uln}, Standard: {certificate.StandardCode}");
+
                     certData.LearnerGivenNames = ilr.GivenNames;
                     certData.LearnerFamilyName = ilr.FamilyName;
                     certData.LearningStartDate = ilr.LearnStartDate;
                     certData.FullName = $"{ilr.GivenNames} {ilr.FamilyName}";
                     certificate.CertificateData = JsonConvert.SerializeObject(certData);
                     certificate.IsPrivatelyFunded = ilr?.FundingModel == PrivateFundingModelNumber;
-
+                    certificate.OrganisationId = organisation.Id;
                     await _certificateRepository.Update(certificate, request.Username, null);
                 }
                 else if (certificate.Status == CertificateStatus.Submitted && certData.OverallGrade == CertificateGrade.Fail)
```

Comment:
New SV Code: Why does the Ilr details not need updating for this 'restart' case

Reply:
This is because when passing a previously failed certificate the other details should remain the same